src/main/java/com/facebook/openwifirrm/optimizers/channel/RandomChannelInitializer.java

```diff
@@ -37,7 +38,10 @@ public class RandomChannelInitializer extends ChannelOptimizer {
 	public static final String ALGORITHM_ID = "random";
 
 	/** The PRNG instance. */
-	private final Random rng = new Random();
+	private Random rng = new Random();
```

Comment:
I would keep the `final` and instantiate it in the constructors, see comment below.